server/notification-providers/signl4.js

```diff
@@ -0,0 +1,85 @@
+const NotificationProvider = require("./notification-provider");
+const axios = require("axios");
+const { Liquid } = require("liquidjs");
+const { UP, DOWN } = require("../../src/util");
+
+class SIGNL4 extends NotificationProvider {
+
+    name = "SIGNL4";
+
+    /**
+     * @inheritdoc
+     */
+    async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
+        let okMsg = "Sent Successfully.";
+
+        try {
+            let data = {
+                heartbeat: heartbeatJSON,
+                monitor: monitorJSON,
+                msg,
+            };
+            let config = {
+                headers: {
+                    "Content-Type": "application/json"
+                }
+            };
+
+            // Source system
+            data["X-S4-SourceSystem"] = "UptimeKuma";
+
+            // Monitor URL
```

Comment:
Please use the following instead
https://github.com/louislam/uptime-kuma/blob/b059c190695b491cd4f10b3c591cbf05887fac85/server/notification-providers/notification-provider.js#L27

Reply:
Ah, great, will be changed.